cwe/cwe.go

```diff
@@ -0,0 +1,111 @@
+package cwe
+
+import (
+	"archive/zip"
+	"bytes"
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"golang.org/x/xerrors"
+)
+
+type CWEConfig struct {
+	url        string
+	retryTimes int
+	cweDir     string
+}
+
+func NewCWEConfig() CWEConfig {
+	return NewCWEWithConfig("https://cwe.mitre.org/data/xml/cwec_latest.xml.zip", filepath.Join(utils.VulnListDir(), "cwe"), 5)
```

Comment:
nit: URL is defined as `const` in other sources.

Reply:
fixed